src/Pelagos/Entity/Dataset.php

```diff
@@ -665,6 +665,36 @@ public function updateAvailabilityStatus()
         }
         $this->setAvailabilityStatus($availabilityStatus);
     }
+    
+    /**
+     * Whether this Dataset is available.
+     *
+     * @return boolean
+     */
+    public function isAvailable() : bool
+    {
+        return $this->availabilityStatus === DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE_REMOTELY_HOSTED or $this->availabilityStatus === DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE;
```

Comment:
For better readability, I think using in_array would be better here (For ex: DatalandUI Controller) and below functions

Reply:
That works for me, in_array always return a bool, and has a strict option. I'll make the changes. Also should we (I) refactor Dataland?